sources/engine/Stride.Graphics/Sprite.cs

```diff
@@ -149,6 +149,7 @@ public Vector4 Borders
         /// <summary>
         /// Gets the value indicating if the image has unstretchable borders.
         /// </summary>
+        [DataMember]
         public bool HasBorders { get; private set; }
```

Comment:
Shouldn't this one be ignored? It's set by `Borders` setter.

Reply:
Same for `SizeInPixels` below which is set by PixelsPerUnit->UpdateSizes()

Reply:
> Shouldn't this one be ignored? It's set by `Borders` setter.

datamembereing it now lets the editor serialize it but its an invlaid case, and is fixed here https://github.com/stride3d/stride/pull/1868

Reply:
Yep, thanks !